pkg/cluster/cluster.go

```diff
@@ -432,6 +432,19 @@ func GetClusterAddOns(connection *sdk.Connection, clusterID string) ([]*AddOnIte
 	}
 	resourceQuotas := resourceQuotasResponse.Items()
 
+	// Get complete list of sku rules
+	skuRulesListResponse, err := connection.AccountsMgmt().V1().SkuRules().List().Send()
+	if err != nil {
+		return nil, fmt.Errorf("Failed to get sku-rules: %v", err)
+	}
+	skuRules := skuRulesListResponse.Items()
+	var skuRulesMap map[string]string
+
+	skuRules.Each(func(skuRule *amsv1.SkuRule) bool {
+		skuRulesMap[skuRule.Sku()] = skuRule.QuotaId()
```

Comment:
This will be a list. One QuotaID can be granted by many SKUs.  For example, you're buying quota for `foobar` and can purchase different size SKUs of `small`, `medium`, `large` that grants 2, 4, and 6 units of `foobar`.